app/pyoxidizer.bzl

```diff
@@ -0,0 +1,36 @@
+# This file defines how PyOxidizer application building and packaging is
```

Comment:
Can you align this to 120 characters please?

Reply:
Done!